gems/dogstatsd-ruby/5.5.0/_test/test.rb

```diff
@@ -0,0 +1,4 @@
+# Write Ruby code to test the RBS.
+# It is type checked by `steep check` command.
+
+require "dogstatsd-ruby"
```

Comment:
Could you add a test case(s) to this file? Such as `Datadog::Statsd.new('localhost', 8125)`.